src/utils.ts

```diff
@@ -141,14 +141,14 @@ export function recoverEthAddressFromPersonalRpcSig(
   rpcSig: string
 ): Buffer {
   // Hash the text the same way web3 does with the weird "Ethereum Signed Message" text
-  const hashed = EthU.hashPersonalMessage(EthU.toBuffer(signedText))
+  const hashed = EthU.hashPersonalMessage(Buffer.from(signedText))
```

Comment:
Do we need to specify the encoding with Buffer.from?

Reply:
Nope, before toBuffer would just do `Buffer.from()` without the encoding if the passed string didn't start with `0x`